src/lib/Components/Bidding/Components/__tests__/Timer-tests.tsx

```diff
@@ -169,7 +169,7 @@ it("shows month, date, and hour adjusted for the timezone where the user is", ()
     </BiddingThemeProvider>
   )
 
-  expect(getTimerLabel(timer)).toEqual("Ends May 14, 1 PM")
+  expect(getTimerLabel(timer)).toEqual("Ends May 14, 1 PM PDT")
```

Comment:
With this change, the timer on the confirm bid page will also show timezone (this felt good/preferable).